[2022-03-18 15:22:23,234] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2011-02-12T04:00:00+00:00 [queued]>
[2022-03-18 15:22:23,255] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2011-02-12T04:00:00+00:00 [queued]>
[2022-03-18 15:22:23,256] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 15:22:23,257] {taskinstance.py:1239} INFO - Starting attempt 2 of 3
[2022-03-18 15:22:23,258] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-03-18 15:22:23,276] {taskinstance.py:1259} INFO - Executing <Task(PythonOperator): get_pbp_data> on 2011-02-12 04:00:00+00:00
[2022-03-18 15:22:23,283] {standard_task_runner.py:52} INFO - Started process 503 to run task
[2022-03-18 15:22:23,288] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'pbp_bucket_ingestion_dag', 'get_pbp_data', 'scheduled__2011-02-12T04:00:00+00:00', '--job-id', '6481', '--raw', '--subdir', 'DAGS_FOLDER/pbp_ingestion_dag.py', '--cfg-path', '/tmp/tmpovbswpk2', '--error-file', '/tmp/tmpqgmndmfe']
[2022-03-18 15:22:23,291] {standard_task_runner.py:77} INFO - Job 6481: Subtask get_pbp_data
[2022-03-18 15:22:23,405] {logging_mixin.py:109} INFO - Running <TaskInstance: pbp_bucket_ingestion_dag.get_pbp_data scheduled__2011-02-12T04:00:00+00:00 [running]> on host 9a1698cd207d
[2022-03-18 15:22:23,532] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=pbp_bucket_ingestion_dag
AIRFLOW_CTX_TASK_ID=get_pbp_data
AIRFLOW_CTX_EXECUTION_DATE=2011-02-12T04:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2011-02-12T04:00:00+00:00
[2022-03-18 15:22:24,006] {python.py:175} INFO - Done. Returned value was: None
[2022-03-18 15:22:24,074] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=pbp_bucket_ingestion_dag, task_id=get_pbp_data, execution_date=20110212T040000, start_date=20220318T152223, end_date=20220318T152224
[2022-03-18 15:22:24,528] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-03-18 15:22:24,902] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
